softioc/softioc.py

```diff
@@ -33,6 +39,14 @@ def iocInit(dispatcher=None):
         dispatcher = cothread_dispatcher.CothreadDispatcher()
     # Set the dispatcher for record processing callbacks
     device.dispatcher = dispatcher
+
+    if enable_pva:
+        dbLoadDatabase('pvxsIoc.dbd', pvxslibs.path.dbd_path, None)
+        ctypes.CDLL(find_dso('pvxslibs.lib.pvxsIoc'), ctypes.RTLD_GLOBAL)
+
+        if registerRecordDeviceDriver(pdbbase):
+            raise RuntimeError('Error registering')
```

Comment:
Very un-Pythonic error handling here!

I suggest moving `registerRecordDeviceDriver` to [imports.py](https://github.com/DiamondLightSource/pythonSoftIOC/blob/9eb29d037d5ec700bcb2616b8b3bfea16b6b64dd/softioc/imports.py) and adding `.errcheck = expect_success` to its definition.

Reply:
Done in 026ed69c20d9a92c729e022fbfbc999aef546d08